server/events.lua

```diff
@@ -90,8 +86,8 @@ AddEventHandler('playerConnecting', OnPlayerConnecting)
 
 RegisterNetEvent('QBCore:server:CloseServer', function(reason)
     local src = source
-    if QBCore.Functions.HasPermission(src, 'admin') or QBCore.Functions.HasPermission(src, 'god') then
-        local reason = reason or 'No reason specified'
+    if QBCore.Functions.HasPermission(src, 'admin') then
```

Comment:
Requires god as well unless we restructure the permission system, god is considered higher than admin.

Reply:
The HasPermission function allows it already even if you check for another permission that isn't god